src/js/components/Calendar/__tests__/__snapshots__/Calendar-test.js.snap

```diff
@@ -131,6 +131,8 @@ exports[`Calendar date 1`] = `
   border: none;
   padding: 0;
   text-align: inherit;
+  font-size: 18px;
+  line-height: 24px;
```

Comment:
Not sure if this is a problem. thoughts? I. see the chromatic changes, so. it can't be good.

Reply:
Agreed. Good job Chromatic!
I think the issue is related to Calendar setting the font size at the Calendar level and expecting that to be inherited. But, the Button is always setting a font size. I'm thinking that `<Button plain />` should not set the font size.

Reply:
Fixed, Chromatic is looking good now.